docs/loadtest_system.md

```diff
@@ -0,0 +1,49 @@
+# Complete load-test system
+
+```
++--------------------------------------------------------------------------------------------------+
+|                                                                                                  |
+|                                                                                                  |
+|                                                                                                  |
+|      +-----------------------------------------------------------+                               |
+|      |                   load-test agent cluster                 |                               |
+|      +-----------------------------------------------------------+                               |
+|      |  +------+ +----------------------------------+            |                               |
+|      |  |      | | +---------------+                |            |                               |
+|      |  |      | | |               |                |            |                               |
+|      |  |  +-------------+  +------v------+  +------v------+     |                               |
++----------->| coordinator |  |             |  |             |     |                               |
+       |  |  +-------------+  |  lt-agent   |  |  lt-agent   |     |                               |
+       |  +->|  lt-agent   |  |             |  |             |     |                               |
+       |     +-------------+  +-------------+  +-------------+     |                               |
+       +-----------------------------------------------------------+                               |
+                                     |                                                             |
+                            +--------v--------+                                                    |
+                            |                 |                                                    |
+                    +-------|      proxy      |-------+                                            |
+                    |       |                 |       |                                            |
+                    |       +-----------------+       |                                            |
+                    |                |                |                                            |
+           +--------|----------------|----------------|--------+                                   |
+           |        |           app  |  cluster       |        |             +------------------+  |
+           +--------|----------------|----------------|--------+             | metrics instance |  |
+           | +------v------+  +------v------+  +------v------+ |             +------------------+  |
+           | |             |  |             |  |             | |------------>|    prometheus    |--+
+           | |app instance |  |app instance |  |app instance | |             +------------------+  |
```

Comment:
The arrow from prometheus to grafana shares the same line that goes to the coordinator. It makes it a bit confusing. It should be a separate arrow, perhaps going directly from the prometheus box to grafana without leaving the shared box.